novawallet/Common/Services/ExtrinsicService/Substrate/ExtrinsicServiceFactory.swift

```diff
@@ -117,7 +117,10 @@ extension ExtrinsicServiceFactory: ExtrinsicServiceFactoryProtocol {
 
         let feeEstimationRegistry = ExtrinsicFeeEstimationRegistry(
             chain: chain,
-            estimatingWrapperFactory: feeEstimatingWrapperFactory
+            estimatingWrapperFactory: feeEstimatingWrapperFactory,
+            chainRegistry: ChainRegistryFacade.sharedRegistry,
```

Comment:
It is hacky and doesn't make sense as we have connection + runtime in the init. Also it is enough to have connection  and runtime in the HydraFlowStateStore and we have them as parameter for fee installer - so, let's try to use them instead of passing chain registry